### PR TITLE
New version

### PR DESCRIPTION
**Pull request by AndersAskeland**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,54 @@
-numpy>=1.14.3
-pandas>=0.22
-h5py>=2.8
-pytorch>=1.2
-tqdm>=4.26
-seaborn>=0.9
-matplotlib>=3.2
\ No newline at end of file
+appnope==0.1.2
```
Comment (thorsteen) [inline requirements.txt]: how did you create this dependency list and is created using on a Linux distro or what kind of bash / pip / ?

File: train.py [hunk 2]
```diff
@@ -105,7 +105,7 @@ def train(args):
                                     'weighted_rmse', 'weighted_mae', 'rmse', 'mse'])
     
     # Epoch (Training) loop
-    for epoch in tqdm(range(args.epochs)):
```
Comment (thorsteen) [inline train.py]: why are are you removing tqdm here?

File: requirements.txt (hunk 1 above)
Comment (AndersAskeland) [reply requirements.txt]: Just a pip freeze from pyenv virtual environment. I don't know where appnope is from. Probably from another dependency.  Still, my virtual env is quite old. I might have some items not used.

File: train.py (hunk 2 above)
Comment (AndersAskeland) [reply train.py]: I first added it as I wanted an outer progress bar, however, it did not work.

File: requirements.txt (hunk 1 above)
Comment (thorsteen) [reply requirements.txt]: I feel like we should keep the old requirements just for simplicity

**Pull request merged**